projects/igniteui-angular/src/lib/combo/combo.component.html

```diff
@@ -22,7 +22,7 @@
             clear
         </igx-icon>
     </igx-suffix>
-    <igx-suffix igxButton="icon" class="igx-combo__toggle-button" igxRipple>
+    <igx-suffix class="igx-combo__toggle-button">
```

Comment:
😁 yup that ripple was kinda odd there 
The button directive I'm not 100% sure though - I think it adds the hover effect and `role=button` which might be beneficial to keep

Reply:
Since the entire select area is clickable, not just the little button, I think it's fine to omit this.